tests/integration/security/egress_gateway_origination_test.go

```diff
@@ -45,6 +45,8 @@ import (
 // TestSimpleTlsOrigination test SIMPLE TLS mode with TLS origination happening at Gateway proxy
 // It uses CredentialName set in DestinationRule API to fetch secrets from k8s API server
 func TestSimpleTlsOrigination(t *testing.T) {
+	// https://issues.redhat.com/browse/OSSM-755
```

Comment:
nit: Put a `FIXME:` prefix to those comments? Perhaps it makes it easier for us in the future to locate them?

Reply:
Done